io/tools/converter.cpp

```diff
@@ -48,10 +48,10 @@
 
 #include <pcl/console/parse.h>
 #include <pcl/io/auto_io.h>
+#include <pcl/io/boost.h>  // for boost::filesystem::path, boost::algorithm::ends_with
```

Comment:
Can we include the boost headers directly?

Reply:
Yes!